common/static/common/templates/discussion/forum-action-vote.underscore

```diff
@@ -2,7 +2,7 @@
     <span aria-hidden="true" class="display-vote is-hidden" >
       <span class="vote-count"></span>
     </span>
-    <a href="#" class="action-button action-vote" role="checkbox" aria-checked="false">
+    <button class="btn-link action-button action-vote" role="checkbox" aria-checked="false">
```

Comment:
Is there JavaScript keeping tabs of the new role here? Meaning, when this is pressed, does `aria-checked` change its value accordingly? Any reason not to use an actual checkbox input type here?